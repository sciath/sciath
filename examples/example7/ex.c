
#include "stdio.h"

int main(void)
{

  printf("@ -- \n");
  printf("@ -- Output from fake application code \n");
  printf("@ -- comment lines \n");
  printf("@ -- \n");
 
  printf("kspits 42\n");
  printf("kspits 58\n");
  printf("kspits 77\n");
  printf("kspits 33\n");

  printf("norm 2.33E+05\n");
  
  return(0);
}
