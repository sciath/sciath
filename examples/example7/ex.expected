@ -- 
@ -- Output from fake application code 
@ -- comment lines 
@ -- 
kspits 42
kspits 58
kspits 77
kspits 33
norm 2.33E+05
@ --
@ -- Output from fake application code
@ -- comment lines
@ --
kspits 42
kspits 58
kspits 21
kspits 33
norm 9.33E+07
